subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/JavaToolchainBuildOperationsIntegrationTest.groovy

```diff
@@ -641,10 +641,31 @@ class JavaToolchainBuildOperationsIntegrationTest extends AbstractIntegrationSpe
         return events
     }
 
+    private <T extends BuildOperationType<?, ?>> List<ProgressEvent> allToolchainEvents() {
```

Comment:
This method is unused. Did you plan on using it somewhere?

Reply:
I am using it actively for debugging when tests fail. It should be removed before merge

Reply:
Done